aten/src/ATen/native/native_functions.yaml

```diff
@@ -37,19 +37,31 @@
   use_c10_dispatcher: full
   variants: function
 
+# Computes the gradient of current tensor w.r.t. graph leaves.
 - func: backward(Tensor self, Tensor? gradient=None, bool keep_graph=False, bool create_graph=False) -> ()
   variants: method
 
+# DEPRECATED. Sets the tensor data held by this `Variable` to be the same as
+# `new_data`.  It requires that `new_data` and `Variable` have compatible tensor
+# type, by checking `_has_compatible_shallow_copy_type(this, new_data)`.
+#
+# This function is deprecated because it doesn't really make sense in a world
+# where Variables *are* Tensors (as opposed to them containing tensors, which
+# is what the previous interpretation was.)
 - func: set_data(Tensor(a!) self, Tensor new_data) -> ()
   use_c10_dispatcher: unboxed_only
   variants: method
 
 - func: data(Tensor self) -> Tensor
   variants: method
 
+# True if this `Variable` is a leaf and thus does not have a `grad_fn`.
```

Comment:
Do we still call them `Variable`s in the comments? Or are we planning on calling everything Tensor in the future?

Reply:
We should call everything Tensor, but that'll be a big codemod so I opted to minimize changes in this PR and have some places refer to things as Variable